app/build.gradle

```diff
@@ -114,3 +114,4 @@ dependencies {
 // This plugin must stay at the bottom
 // https://developers.google.com/android/guides/google-services-plugin
 apply plugin: 'com.google.gms.google-services'
+apply plugin: 'kotlin-android-extensions'
```

Comment:
I know others have tried to add this plugin in the past.  I would move it up to the other plugins if you are using it.

Reply:
Was there a particular reason it hasn't been added in the past?

I have moved it up with the other plugins in the latest commit.

Reply:
I'm not sure... it wasn't me against it.